masterlist.yaml

```diff
@@ -798,8 +809,77 @@ plugins:
     after:
       - 'Bashed Patch.*\.esp'
   - name: 'Ordinator - Perks of Skyrim.esp'
+    after: 
+      - 'Phenderix Magic Evolved.esp'
+      - 'Phenderix Magic World.esp'
+      - 'Undeath.esp'
     inc:
       - 'ScopedBows_EagleEyeTweak.esp'
+      - 'PerkusMaximus_Mage.esp'
+      - 'PerkusMaximus_Master.esp'
+      - 'PerkusMaximus_Thief.esp'
+      - 'PerkusMaximus_Warrior.esp'
+      - 'PerkusMaximus_CombatPlus.esp'
+    msg:
+      - type: warn
```

Comment:
These messages look like they're a good candidate for having a common message to re-use, e.g. something like [this in the Fallout 4 masterlist](https://github.com/loot/fallout4/blob/v0.10/masterlist.yaml#L110).

Reply:
Yes, would be better to cut down on repeating same entries. I'll have a try at making it better.